index.js

```diff
@@ -1,286 +1,27 @@
-
-                let foundStr = joinAndCommas(found.map(w => w.word))
-                react(bot, emoji, message, noop)
-                let poser = found.map(w => w.addedBy ? `<@${w.addedBy}>` : '').join(' ')
-                bot.reply(message, {
-                    username: 'bingo',
-                    text: `Bingo! <@${message.user}> said ${foundStr}! There are ${words.length || "no"} bingo words left to discover! ${poser}`,
-                    icon_emoji: `:${emoji}:`,
-                    attachments: found.filter(w => w.gif).map(attachmentGif)
-                })
-            }
-
-            if (alreadyFound.length) {
-                let foundStr = joinAndCommas(alreadyFound.map(w => {
-                    if (w != undefined) {
-                        return w.word
-                    }
-                }));
-
-                bot.reply(message, {
-                    username: 'bingo',
-                    text: `The following words were already bingoed: ${foundStr}`,
-                    icon_emoji: `:${emoji}:`
-                })
-            }
-        }
-    }
-} 
-
-function handleAdd(bot, message) {
-    //We need to extract single words and quoted words
-    message.text.match(/(?:"([^"]+)"|([^"\s]+))/g).splice(1).forEach(w => {
-        if (addWord(w, message.user)) {
-            bot.reply(message, {
-                text: 'Added ' + w,
-                icon_emoji: `:${emoji}:`
-            });
-            react(bot, 'thumbsup', message, noop)
-        }
-    })
-}
-
-function handleCheat(bot, message) {
-    react(bot, 'wink', message, noop)
-    bot.reply(message, words.map(w => w.word).join(', '))
-}
-
-function exitHandler() {
-    persistence.save(words, foundWords, function (err) {
-        if (err) {
-            throw err;
-        } else {
-            console.log('Wordlist and found words saved successfully.');
-            process.exit(0);
-        }
-    });
-    if (typeof bot.destroy === 'function') {
-        bot.destroy(bot);
-    }
-}
-
-//---------------------------
-// Event handler
-//---------------------------
-
-controller.hears('add', ['direct_message'], handleAdd)
-
-controller.hears('cheat', ['direct_message'], handleCheat)
-
-controller.hears(['hello', 'hi', 'introduce'], ['direct_message', 'direct_mention'], introduce)
-
-controller.on('ambient', handleAmbient)
-
-controller.on('bot_channel_join', introduce)
-
-//---------------------------
-
-//catches ctrl+c event
-process.on('SIGINT', exitHandler);
-
-// connect the bot to a stream of messages
-controller.spawn({
-    token: process.env.SLACK_TOKEN
-}).startRTM((err, b, payload) => {
-    if (err) throw err
-    botUid = payload.self.id
-    bot = b;
-})
-
-loadWords();
-
-// Hack: Make functions available for testing
-exports.containsValidSentences = containsValidSentences;
-exports.handleAmbient = handleAmbient
+var botController = require('./bot'),
+    wordManager = require('./wordManager');
+
+// function exitHandler() {
```

Comment:
You leave the code commented inside to remind us to use a DB? Otherwise I think we can remove it.